resources/cli-configs/train-subdomains-paper.yaml

```diff
@@ -0,0 +1,16 @@
+- lat-min: 35
```

Comment:
Can this file then be deleted, or is it potentially used somewhere else?

Reply:
That config is handy to refer to. It gets mentioned in the root readme or document on reproducing Guillaumin (2021). Nicer than writing out in text, since a YAML file is human+machine readable (and we load it for training).